lib/pivotal-tracker/proxy.rb

```diff
@@ -0,0 +1,29 @@
+class BasicObject #:nodoc:
```

Comment:
@wlffann @tmikeschu Yes! Thanks for tackling this. It seems like a pretty slick way to handle objects.